### ISSUE TITLE
No license information in source code

### ISSUE THREAD

**Issue by holtkamp** [Rhincodon/utm-url-parser#1]
Just found this lib and it looks useful to use / contribute. However I could not find license information in the .php source code... Might be useful to include it ;)

**Comment by Rhincodon** [comment]
Hi, there is MIT License in [repo](https://github.com/Rhincodon/utm-url-parser/blob/master/LICENSE.md).

**Comment by holtkamp** [comment]
yes, I noticed, but not in the PHP files itself... I thought this was a
requirement as well. Feel free to close this issue if I am wrong ;)

Hi, there is MIT License in repo
https://github.com/Rhincodon/utm-url-parser/blob/master/LICENSE.md.
